punish/punish.py

```diff
@@ -1118,6 +1178,7 @@ def execute_queue_event(self, *args) -> bool:
         updating_case = False
         case_error = None
         mod = self.bot.get_cog('Mod')
+        remove_role_list = self.json[server.id]["REMOVE_ROLE_LIST"]
```

Comment:
Get this list later (just above your existing `if member.id not in self.json[server.id]:` for example), and use
```py
remove_role_list = self.json[server.id].get("REMOVE_ROLE_LIST", [])
```
to handle missing keys.